test/SymbolGraph/Symbols/SkipProtocolImplementations.swift

```diff
@@ -45,3 +49,15 @@ public struct SomeStruct: SomeProtocol {
     /// Local docs
     public func otherFunc() {}
 }
+
+// Make sure that protocol conformances added in extensions don't create bogus symbol relationships (rdar://107432084)
```

Comment:
can you add a test that checks that we are still emitting it for one that has it's own doc comment?

Reply:
Pushed a commit that adds this to the test.